EurorackPanelDesigner/EurorackPanelDesigner.py

```diff
@@ -19,358 +27,237 @@ def __init__(self):
-                        draw_SVG_line( (x1, y1), (x2, y2), grp)
-                        #Vertical Line
-                        x1=self.unittouu(str(RightHoles) + self.options.unit)
-                        y1=self.unittouu(str(BottomHoles+1.6) + self.options.unit)
-                        x2=self.unittouu(str(RightHoles) + self.options.unit)
-                        y2=self.unittouu(str(BottomHoles-1.6) + self.options.unit)
-                        draw_SVG_line( (x1, y1), (x2, y2), grp)
-                    #Top Right Centers
-                        #Horizontal Line
-                        x1=self.unittouu(str(RightHoles-1.6) + self.options.unit)
-                        y1=self.unittouu(str(TopHoles) + self.options.unit)
-                        x2=self.unittouu(str(RightHoles+1.6 ) + self.options.unit)
-                        y2=self.unittouu(str(TopHoles ) + self.options.unit)
-                        draw_SVG_line( (x1, y1), (x2, y2), grp)
-                        #Vertical Line
-                        x1=self.unittouu(str(RightHoles) + self.options.unit)
-                        y1=self.unittouu(str(TopHoles+1.6) + self.options.unit)
-                        x2=self.unittouu(str(RightHoles) + self.options.unit)
-                        y2=self.unittouu(str(TopHoles-1.6) + self.options.unit)
-                        draw_SVG_line( (x1, y1), (x2, y2), grp)
-
-        if oval == True:
-
-            # Oval: (is a square with rounded corners)
-
-
-            cxl = self.unittouu(str(LeftHoles - 2.75) + self.options.unit)
-            cxr = self.unittouu(str(RightHoles - 2.75) + self.options.unit)
-            cyb = self.unittouu(str(BottomHoles - 1.6) + self.options.unit)
-            cyt = self.unittouu(str(TopHoles - 1.6) + self.options.unit)
-
-            ovalh = self.unittouu(str(3.2) + self.options.unit)
-
-            ovalw = self.unittouu(str(5.5) + self.options.unit)
-            ovals = self.unittouu(str(1.6) + self.options.unit)
-
-            #Bottom Left
-            draw_SVG_square((ovalw,ovalh), (cxl,cyb), (ovals,0), grp)
-            #Top Left
-            draw_SVG_square((ovalw,ovalh), (cxl,cyt), (ovals,0), grp)
+                    # Bottom Right Centers - Horizontal Line
+                    self.draw_SVG_line( (rightH-holeR+gap, bottomH), (rightH+holeR-gap, bottomH), group)
+                    # Vertical Line
+                    self.draw_SVG_line( (rightH, bottomH+holeR-gap), (rightH, bottomH-holeR+gap), group)
+                    # Top Right Centers - Horizontal Line
+                    self.draw_SVG_line( (rightH-holeR+gap, topH), (rightH+holeR-gap, topH), group)
+                    # Vertical Line
+                    self.draw_SVG_line( (rightH, topH+holeR-gap), (rightH, topH-holeR+gap), group)
+
+        else: # oval == True
+            # Oval Holes: (a square with rounded corners)
+            oval_size = 5.5  # 3.2mm hole. Oval is 5.5mm across
```

Comment:
Pretty sure oval dimension can now be adjusted too in this one place.. But not tried it